tools/find_deprecated_escaped_characters.py

```diff
@@ -0,0 +1,69 @@
+#! /usr/bin/env python
+"""
+Look for escape sequences deprecated in Python 3.6.
+
+Python 3.6 deprecates a number of non-escape sequences starting with `\` that
+were accepted before. For instance, '\(' was previously accepted but must now
```

Comment:
Indeed, should be a raw string. Doesn't show up normally because when run internal deprecations are ignored.